Client/src/Features/UI/uiSlice.js

```diff
@@ -15,7 +15,7 @@ const initialState = {
 	sidebar: {
 		collapsed: false,
 	},
-	mode: "light",
+	mode: window?.matchMedia?.('(prefers-color-scheme: dark)')?.matches? "dark": "light",
```

Comment:
Not sure what happened, but the formatting is not that clear without the space between the first part of the ternary and the question mark. Would you mind extracting the ternary from the object to increase legibility? I suggest saving it to a variable with a name like initialMode or something like that

Reply:
@marcelluscaio I have made that change